packages/apps/testbench-app/src/components/ItemTable.tsx

```diff
@@ -42,6 +42,7 @@ export const ItemTable = <T extends EchoReactiveObject<any>>({ schema, objects =
         stickyHeader
         border
         getScrollElement={() => containerRef.current}
+        estimatedRowHeight={29}
```

Comment:
I think our standards are 32 (fine) and 40 (course) re density.
These values are defined in the theme @thure ?

Reply:
Ah, if these values are coming from the DensityProvider then I should be able to use that directly instead of supply a value as a prop.

Reply:
Yes, though I can’t speak to whether they apply here specifically.

Reply:
@ZaymonFC if the row height is being used here, you may need to make a ternary operation on the provider’s value — we don’t define the size in pixels, (rather it’s `rem`), so it will differ between mobile and desktop as well as between the different density values.